### ISSUE TITLE
Custom Fields in Universal Flexicontent Module aren't parsed/retrieved correctly

### ISSUE THREAD

**Issue by ldoliv** [FLEXIcontent/flexicontent-cck#667]
Hello,

Don't know if the title description is corret but, I updated flexicontent from 3.1.0-rc2 to 3.1.1 and fields I had in custom fields aren't showing anymore, only the first one:
![image](https://cloud.githubusercontent.com/assets/6535818/20301805/c00d6284-ab1c-11e6-8304-a20b97026c6d.png).
I have a bunch of universal flexicontent modules, It will be too much trouble to edit each one and add the fields again, as I would have to check the code and see what fields I need again. Could you please fix and update 3.1.1 ?

**Comment by ggppdk** [comment]
The old values should work, see here:
https://github.com/FLEXIcontent/flexicontent-cck/blob/master/admin/elements/fields.php#L240

Mybe space check is missing:
Try adding whitespace check,```php

Replace:
```php
$values = preg_split("/[\|,]/", $values);
```

with:
```php
$values = preg_split("/[\s]*[\|,][\s]*/", $values);
```

Also if a fieldname is not found, it will be skipped by the updated "fields" element

**Comment by ldoliv** [comment]
Yes, that fixed it. Thank you. Please add this in the next release.

**Comment by ggppdk** [comment]
Fixed with: eb5443a
